package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "test": "mocha-chrome test/index.test.html"
+    "test": "tools/mocha-chrome-wrapper */test/index.test.html"
```

Comment:
What you've done here seems reasonable to me. My thoughts on reading for reference:

I thought maybe `mocha` or `mocha-chrome` could sort this out for us (globbing html files, running them in a batch) but I guess it doesn't do anything fancy test-html-wise, all I see is this:

https://mochajs.org/#the-test-directory

So then we could debate putting `index.test.html` in the test directory vs. the one above it. What you have here seems preferred since we won't have to have the `test/` prefix on all of the files the html file references.

Reply:
I was also hoping that something else would do the globbing but I didn't find anything. When we do we should rip this out.